stacktrace.sh

```diff
@@ -0,0 +1,16 @@
+echo "STARTING"
+
+sudo perf record -p $1 -F 100 -g -- sleep 60 &
```

Comment:
no code in this repo can presume upon or request superuser privileges

Reply:
Sorry this shouldn't be in this pr.

Reply:
Fixed https://github.com/steemit/condenser/pull/2318/commits/984415efbe8f3ad91bb3515655d8d05ad4e7d1bc